app/models/resource_request.rb

```diff
@@ -1,6 +1,10 @@
 # frozen_string_literal: true
 
 class ResourceRequest < ApplicationRecord
+  attr_accessor :create_callback
```

Comment:
As a follow up to my other comment about `create_callback`, the naming is also a little problematic because the attr name doesn't indicate whether it's before create or after create.  (But also, it's really after commit!)